doc/md/fundamentals/5-control-flow/1-basic-control-flow.md

```diff
@@ -139,58 +154,68 @@ If the block produces a non-`()` result, the `break` can include a value. Labels
 When a labeled block runs, it evaluates the block to produce a result. Labels don’t change how the block executes but enable early exits from the block using a `break` to that label. If the type is not `()` those breaks must have an argument, to use as the result of the labelled expression. Just as `return` exits a function early with a result, `break` exits its label early with a result.
 
 ```motoko no-repl
```

Comment:
These examples are a bit weird. Rather than suggest the changes, I'll open a branched PR to make my edits for you to approve and merge into this branch